x-pack/plugins/cases/public/containers/use_find_case_user_actions.test.tsx

```diff
@@ -87,17 +88,16 @@ describe('useGetCaseUserActions', () => {
 
   describe('getProfileUids', () => {
     it('aggregates the uids from the createdBy field of a user action', async () => {
-      jest
-        .spyOn(api, 'getCaseUserActions')
-        .mockReturnValue(
-          Promise.resolve([
-            getUserAction('pushed', Actions.add, { createdBy: { profileUid: '456' } }),
-          ])
-        );
+      jest.spyOn(api, 'findCaseUserActions').mockReturnValue(
+        Promise.resolve({
+          ...findCaseUserActionsResponse,
```

Comment:
nit: Another option for these tests is to just create an object that has:

```
{
  total...,
  page...,
  perPage...,
}
```

I think that's why we're including the `findCaseUserActionsResponse`? That way we don't make it harder to determine which user actions are the ones that the test is actually using.